### ISSUE TITLE
UI improvement (2) : remove conflict between elements

### ISSUE THREAD

**Issue by wildan3105** [wildan3105/github-langs#9]
**Current behavior:**
It's so messy. Look at the image below : 
![ui](https://user-images.githubusercontent.com/7030099/31361376-4aa2a6bc-ad7d-11e7-9550-189364688bdb.png)

**Expected behavior:**
All elements displayed well. The search bar, fork me button, message, the chart of languages and footer. Well displayed with no conflicts. One last thing : just remove Language(s) that following by the details because it already displayed by the chart.js

**Comment by Ceciceciceci** [comment]
I'll try this! 
Is it okay to get rid of data-languages tag or is is necessary for it to be there?

**Comment by wildan3105** [comment]
Good @Ceciceciceci 
No, just get rid these element : 
```html
<p><h4 style="display:{{showLanguage}}"> Language(s) : </h4>
{{#each languages}} {{@key}}: {{this}}, {{/each}} </p>
```
And make the footer not fixed.

**Comment by Ceciceciceci** [comment]
Oh! I made it so you can still have it, but just not displayed, hope that is okay!
I made a pull request :')

**Comment by Ceciceciceci** [comment]
Fixed up the conflict! Thanks.

**Comment by wildan3105** [comment]
Okay! Will check it first @Ceciceciceci

**Comment by wildan3105** [comment]
Closed by #27
